### ISSUE TITLE
Holding "Shift" on Windows 10 Apps Does not show Highlight

### ISSUE THREAD

**Issue by m3taphysics** [digitalcreations/MaxTo#169]
I am using windows 10, holding "shift" on some older applications shows the potential docking position. However, applications like Chrome and MS Edge do not show their target docking locations.

Is this a theme problem? How can we fix this. GREAT tool by the way!

**Comment by vegardlarsen** [comment]
It could very well be a problem with how Chrome and Edge signal to windows that they are being moved around. Could it be that the preview window is hidden behind something else? I have seen that happen before.

**Comment by vegardlarsen** [comment]
I have just fixed an issue that may have had an effect here, but it may not be the exact cause of this issue. UWP apps do not have an icon; which means that MaxTo cannot find an icon to display for the preview window it display when dragging while holding Shift.

This caused MaxTo to retry getting the icon when the next movement message came, raising exceptions (which are slow to handle). This meant that if you let the Shift key go before all of the messages were processed, the window would not be resized into the region. This could take multiple seconds.

I've fixed this so it will only check for an icon once (when starting the drag operation).

However, this only affected Edge. Chrome seemed to work well. It did however affect all Windows Store apps.
